src/components/new-request/business-lookup.vue

```diff
@@ -13,15 +15,12 @@
       append-icon="mdi-magnify"
       autocomplete="chrome-off"
       autofocus
-      filled
       hint="Search by name, incorporation or registration number of an existing business"
-      item-text="name"
-      item-value="identifier"
```

Comment:
Since you've removed `item-value`, does this still return the correct value for a selected item?

Reply:
I answered my own question by creating a NR and looking at its data:

![image](https://github.com/bcgov/namerequest/assets/10002889/5ec183f5-f422-4c56-a03e-6424df6df34a)

So: great!